blocks/cards/cards.js

```diff
@@ -5,14 +5,32 @@ export default function decorate(block) {
   const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
     const li = document.createElement('li');
+    // li.classList.add('ethos-2-theme')
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
-      if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      if (div.children.length === 1 && div.querySelector('img')) {
+        div.className = 'card-image';
+      } else {
+        div.className = 'card-container';
+      }
+      div.querySelectorAll('h2,h3,h4,h5')
+        .forEach((e) => {
+          e.classList.add('card-title', 'heading-5');
```

Comment:
I think this should be `e.className = 'card-title', 'heading-5'` , to remove any other class that might have been applied through global styling.

Reply:
not sure about this. If i do like you proposed, the bold aspect is lost. Of course, this can be fixed via the doc file, but other characteristics may be lost as well.